hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/OpProvider.java

```diff
@@ -44,15 +43,14 @@ public interface OpProvider {
     ResponseCodeEnum[] STANDARD_PERMISSIBLE_PRECHECKS = {
         OK,
         BUSY,
-        INVALID_SIGNATURE,
         DUPLICATE_TRANSACTION,
         INVALID_PAYER_SIGNATURE,
         INSUFFICIENT_PAYER_BALANCE,
         PLATFORM_TRANSACTION_NOT_CREATED
     };
 
     ResponseCodeEnum[] STANDARD_PERMISSIBLE_OUTCOMES = {
-        SUCCESS, LIVE_HASH_NOT_FOUND, INVALID_SIGNATURE, INSUFFICIENT_PAYER_BALANCE, UNKNOWN
```

Comment:
Probably we should not remove INVALID_SIGNATURE from the standard prechecks and outcomes

Reply:
There are 50/50 cases where we don't want this status/where we want it, so i've added `customOutcomes`, which contains `INVALID_SIGNATURE` to the operations that require it.
This way it's more future proof.